File: vmc/beginner/l2.5/AverageOfSquares.c
#include<stdio.h>
#include<math.h>

float average_of_squares0(int n) //starts from 0 to n-1
{
    float sum = 0;
    for (int i = 0; i < n; i++)
    {
        sum += powf(i,2);
    }
    sum = sum/n;
    return sum;
}

float average_of_squares1(int n) //starts from 1 to n
{
    float sum = 0;
    for (int i = 1; i <= n; i++)
    {
        sum += powf(i,2);
    }
    sum = sum/n;
    return sum;
}

int main()
{
    printf("The average squares of 0 to 4 is %f \n",average_of_squares0(5));
    printf("The average squares of 0 to 4 is %f \n",average_of_squares1(4));
    return 0;
}

File: vm/beginner/l2.5/AverageOfSquares.py

#In it create a function 'average_of_squares0(n) which prints out the  average of squares from 0 to n-1.
#   Use it to print out  average_of_squares0(5) (the output should make the output self-explanatory). 
def average_of_squares0(n): #starts from 0 to n-1
    sum = 0
    for i in range(n):
        sum = sum + i**2
    sum = sum/n
    return sum
print("The average squares of 0 to 4 is",average_of_squares0(5))

def average_of_squares1(n): #starts from 1 to n
    sum = 0
    for i in range(1,n+1):
        sum = sum + i**2
    sum = sum/n
    return sum
print("The average squares of 0 to 4 is",average_of_squares1(4))